tests/Oryx.BuildImage.Tests/OryxCommandTest.cs

```diff
@@ -6,14 +6,36 @@
 using Xunit;
 using Xunit.Abstractions;
 using Microsoft.Oryx.BuildScriptGenerator.Node;
-using Microsoft.Oryx.Common;
 using Microsoft.Oryx.BuildScriptGenerator.Php;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
 
 namespace Microsoft.Oryx.BuildImage.Tests
 {
-    public class ExecCommandTest : SampleAppsTestBase
+    public class OryxCommandTest : SampleAppsTestBase
     {
-        public ExecCommandTest(ITestOutputHelper output) : base(output) { }
+        public OryxCommandTest(ITestOutputHelper output) : base(output) { }
+
+        [Fact]
+        public void Build_UsesCwd_WhenNoSourceDirGiven()
+        {
+            // Act
+            var result = _dockerCli.Run(new DockerRunArguments
+            {
+                ImageId = Settings.BuildImageName,
+                CommandToExecuteOnRun = "oryx",
+                CommandArguments = new[] { "build" },
+                WorkingDirectory = "/tmp"
+            });
+
+            // Assert
+            RunAsserts(
+                () =>
+                {
+                    Assert.Contains("Error: Could not detect", result.StdErr);
```

Comment:
Instead of an error case, we could just build a sample app of any platform and test for sucess